lib/events/api.go

```diff
@@ -252,6 +252,11 @@ const (
 	X11ForwardSuccess = "success"
 	X11ForwardErr     = "error"
 
+	// X11 forwarding channel open event
+	X11ForwardChannelEvent   = "x11-forward-channel"
+	X11ForwardChannelSuccess = "success"
```

Comment:
Can you just use `X11ForwardSuccess` and `X11ForwardErr`? Why do we need two sets of constants with the same values?